docs/source/jupyter/ontology-interface.ipynb

```diff
@@ -327,52 +814,81 @@
      "name": "stdout",
+     "output_type": "stream",
      "text": [
-      "\nYou can get the argument name of an attribute. The argument name is used when instantiating CUDS objects\nage\n\nYou can get the datatype of attributes\nhttp://www.w3.org/2001/XMLSchema#integer\n\nYou can use the attribute to convert values to the datatype of the attribute\n10\n"
+      "\n",
+      "You can get the argument name of an attribute. The argument name is used as keyword argument when instantiating CUDS objects.\n",
+      "age\n",
+      "\n",
+      "You can get the datatype of attributes\n",
+      "http://www.w3.org/2001/XMLSchema#integer\n",
+      "\n",
+      "You can use the attribute to convert values to the datatype of the attribute\n",
+      "<class 'int'> 10\n",
+      "\n",
+      "And likewise to convert values to the python basic type associated with the datatype of the attribute.\n",
+      "<class 'str'> 5\n"
      ]
     }
    ],
    "source": [
     "print(\"\\nYou can get the argument name of an attribute. \"\n",
-    "      \"The argument name is used when instantiating CUDS objects\")\n",
+    "      \"The argument name is used as keyword argument when instantiating CUDS objects.\")\n",
     "print(city.age.argname)\n",
     "\n",
     "print(\"\\nYou can get the datatype of attributes\")\n",
     "print(city.age.datatype)\n",
     "\n",
     "print(\"\\nYou can use the attribute to convert values \"\n",
     "      \"to the datatype of the attribute\")\n",
-    "print(city.age.convert_to_datatype(\"10\"))"
+    "result = city.age.convert_to_datatype(\"10\")\n",
+    "print(type(result), result)\n",
+    "\n",
+    "print(\"\\nAnd likewise to convert values to the python basic type \"\n",
+    "      \"associated with the datatype of the attribute.\")\n",
+    "result = city.name.convert_to_basic_type(5)\n",
+    "print(type(result), result)"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## CUDS\n",
+    "## Creating CUDS using ontology classes\n",
     "\n",
-    "You can use ontology classes to create CUDS objects."
+    "You can call ontology classes to create CUDS objects."
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 11,
+   "execution_count": 23,
    "metadata": {
     "tags": []
    },
    "outputs": [
     {
-     "output_type": "stream",
      "name": "stdout",
-     "text": "\nYou can instantiate CUDS objects using ontology classes\ncity.Citizen: 1714a778-5943-4611-af3a-e3b0cb32693d\nTake a look at api_example.py for a description of the CUDS API\n\nYou can check if a CUDS object is an instace of a ontology class\nTrue\nTrue\n\nYou can get the ontology class of a CUDS object.\ncity.Citizen\n"
+     "output_type": "stream",
+     "text": [
+      "\n",
+      "You can instantiate CUDS objects using ontology classes\n",
+      "city.Citizen: e0947100-9c40-415f-92c8-a86b796dbb01\n",
+      "Take a look at api_example.py for a description of the CUDS API\n",
```

Comment:
Perhaps this line should be out of the code block and be put as a plain text about it.

Reply:
I thought of a solution, coming in the next commit.